12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,90 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+  #amir@ubuntu:~/wildcards$ find ./ -name "[[:lower:]]*.*"
```

Comment:
Also the task was to find files that start with a lowercase letter. Not all files in unix have `.` (period) sign in them, for example file `/etc/shells`, `/bin/ls` etc. have no extension. 
It's a normal thing on Unix systems not to have extension for every file.

Searching for `[[:lower:]]*.*` only searches for files with extensions, but on Unix we prefer the `[[:lower:]]*` to search for all files starting with a lowercase letter, with or without file extensions